WordPress/src/main/java/org/wordpress/android/ui/jetpack/scan/ScanViewModel.kt

```diff
@@ -60,6 +63,24 @@ class ScanViewModel @Inject constructor(
         }
     }
 
+    private fun startScan() {
+        launch {
+            startScanUseCase.startScan(site)
+                .flowOn(bgDispatcher)
```

Comment:
🔍 Nitpick: We might want to set the dispatcher in the usecase so no-one can invoke it on the main thread. Wdyt?

Reply:
In 1c57465